apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2.Snippets/TranslationClientSnippets.cs

```diff
@@ -22,21 +22,21 @@ namespace Google.Cloud.Translation.V2.Snippets
     public class TranslationClientSnippets
     {
         
-        public void TranslateTextNmtDefaultModel()
+        public void TranslateTextPbmtDefaultModel()
         {
             // Sample: TranslateTextNmtDefaultModel
-            TranslationClient client = TranslationClient.Create(model: TranslationModel.NeuralMachineTranslation);
+            TranslationClient client = TranslationClient.Create(model: TranslationModel.PhraseBasedMachineTranslation);
             TranslationResult result = client.TranslateText("It is raining.", LanguageCodes.French);
             Console.WriteLine($"Result: {result.TranslatedText}; detected language {result.DetectedSourceLanguage}");
             // End sample
         }
 
-        public void TranslateTextNmtOverrideModel()
+        public void TranslateTextPbmtOverrideModel()
         {
             // Sample: TranslateTextNmtOverrideModel
```

Comment:
Yes indeed, thank you! Hopefully AppVeyor would have caught that - I'm going to let it go red just to prove that, then fix it, push and wait for it to go green.